go.mod

```diff
@@ -3,16 +3,17 @@ module github.com/filecoin-project/dagstore
 go 1.16
 
 require (
-	github.com/filecoin-project/go-indexer-core v0.2.4
+	github.com/gopherjs/gopherjs v0.0.0-20190430165422-3e4dfb77656c // indirect
```

Comment:
Looks like this might have been imported by accident?

Reply:
fixed.